### DISCUSSION TITLE
[Web] Quill Editor asking for clipboard permission.

### DISCUSSION THREAD

**Discussion by franciscojavierlf** [singerdmx/flutter-quill#302] [Web]
I have only tested on [Web], but each time I render a QuillEditor I get prompted for clipboard permissions, and it's a little annoying. Any way of disabling this? Maybe I'm missing something.

**Comment by andrewjallenuoft** [comment]
Struggling with this issue as well. Would be great to get a fix on this.

**Comment by NicolasDionB** [comment]
Also have this problem. It's even more annoying when you put the editor in readonly mode! Just doesn't make any sense...

I just created an issue from this discussion. 😉
